ethereum/erc20/tokens.sql

```diff
@@ -181,6 +181,7 @@ COPY erc20.tokens (contract_address, symbol, decimals) FROM stdin;
 \\xc4cb5793bd58bad06bf51fb37717b86b02cbe8a4	CREDIT	18
 \\x4e0603e2a27a30480e5e3a4fe548e29ef12f64be	CREDO	18
 \\x158079ee67fce2f58472a96584a73c7ab9ac95c1	cREP	8
+\\xcbc1065255cbc3ab41a6868c22d1f1c573ab89fd CRETH2  18
```

Comment:
These all need to be tab separated

Reply:
@mewwts Thanks for the clarification. I've updated the PR.
Let me know if there's anything else I need to change.